object/src/mem.rs

```diff
@@ -618,34 +618,55 @@ where
     /// Insert a data element to the object, replacing (and returning) any
     /// previous element of the same attribute.
     pub fn put_element(&mut self, elt: InMemElement<D>) -> Option<InMemElement<D>> {
-        self.entries.insert(elt.tag(), elt)
+        self.len = Length::UNDEFINED;
```

Comment:
I suppose that this is already done inside the `map`, right?
```suggestion
```

Reply:
right ...